wagtail/wagtailadmin/templates/wagtailadmin/account/account.html

```diff
@@ -40,6 +40,17 @@
                     </small>
                 </li>
             {% endif %}
+                <li class="row row-flush">
+                    <div class="col6">
+                        <a href="{% url 'wagtailadmin_account_language_preferences' %}" class="button button-primary">{% trans "Language preferences" %}</a>
+                    </div>
+
+                    <small class="col6">
+                        {% trans "Choose which language do you want to use in this admin." %}
```

Comment:
"Choose which language do you want to use in this admin" -> "Choose the language you want to use here"